### DISCUSSION TITLE
Habemus MIT no React

### DISCUSSION THREAD

**Discussion by RatoX** [frontendbr/forum#737] [Discussão]
https://code.facebook.com/posts/300798627056246/relicensing-react-jest-flow-and-immutable-js/

Destaque para: 
> we don't want to hold back forward progress for nontechnical reasons.

**Comment by glauberramos** [comment]
ótima notícia pra comunidade

**Comment by ghost** [comment]
Isso só prova o poder de uma comunidade unida! Ótima noticia.
